### DISCUSSION TITLE
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by Faust-Wang** [community/community#52592] [GitHub Education]
When I tried to apply forGitHub students account, I received the following message：
“There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact?tags=education-support) for more information.”
I submitted the work order on github support, and 16 days later there is still no admin working on the issue.

**Comment by wanng-ide** [comment]
Me too!!!!

**Comment by samcao416** [comment]
Me too, have you solved this issue?

**Comment by zhangchi25806** [comment]
Same here and no anwsers yet
